pyproject.toml

```diff
@@ -7,42 +7,34 @@ max-branches=50
 max-nested-blocks=10
 max-statements=200
 
-[tool.isort]
-profile = "black"
-src_paths = ["src", "tests/ci", "tests/sqllogic"]
-
 [tool.pylint.FORMAT]
 #ignore-long-lines = (# )?<?https?://\S+>?$
 
 [tool.pylint.'MESSAGES CONTROL']
           # pytest.mark.parametrize is not callable (not-callable)
 disable = '''
-          pointless-string-statement,
-          line-too-long,
           missing-docstring,
           too-few-public-methods,
           invalid-name,
           too-many-arguments,
-          keyword-arg-before-vararg,
           too-many-locals,
           too-many-instance-attributes,
           cell-var-from-loop,
           fixme,
           too-many-public-methods,
           wildcard-import,
-          unused-wildcard-import,
-          singleton-comparison,
-          not-callable,
           redefined-outer-name,
           broad-except,
           bare-except,
-          no-else-return,
-          global-statement,
-          f-string-without-interpolation,
-          consider-using-with,
```

Comment:
this is excessive check for ci scripts

Reply:
This is a must-have practice to keep the resources private in the scope of functions. I found the only single place where it complained, and in the place the check was abused https://github.com/ClickHouse/ClickHouse/pull/70013/files#diff-0397dcb4f24f20fe4b2d7925174332bb6e84f670da123876c12544872e5b1c3fR251